<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-0CV2FCB" ignored="0" name="Test" tests="6" failures="1" timestamp="2021-06-27T02:14:25 IST" time="0.12" errors="0">
  <testcase name="createProject" time="0.014" classname="com.actitime.testscript.ProjectModule"/>
  <testcase name="deleteProject" time="0.002" classname="com.actitime.testscript.ProjectModule"/>
  <testcase name="createCustomer" time="0.003" classname="com.actitime.testscript.CustomerModule"/>
  <testcase name="deleteCustomer" time="0.003" classname="com.actitime.testscript.CustomerModule"/>
  <testcase name="createTask" time="0.002" classname="com.actitime.testscript.TaskModule"/>
  <testcase name="deleteTask" time="0.011" classname="com.actitime.testscript.TaskModule">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.actitime.testscript.TaskModule.deleteTask(TaskModule.java:20)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- deleteTask -->
</testsuite> <!-- Test -->
